project/project.go

```diff
@@ -30,7 +30,7 @@ func Load(projectPath string) (*Project, error) {
 
 	projectBaseName := filepath.Base(absProjectPath)
 
-	projectProxy, projectProxyErr := proxy.GetProxy(projectPath)
+	projectProxy, projectProxyErr := proxy.Get(projectPath)
```

Comment:
I've resigned myself to reusing `err` over and over if you want to change it here, or I can...